src/components/pages/blogs/blog/components/TableTemplate.tsx

```
@@ -0,0 +1,76 @@
+const TableTemplate: React.FC<ITableTemplate> = ({ content }) => {
+  return (
+    <div className="border-t pt-5">
+      <h2 className="font-black text-[18px] lg:text-[25px] leading-[48px]">
+        {content.title}
+      </h2>
+
+      {content.topDescription?.split("\n").map((item, index) => (
+        <p
+          key={index}
+          className="font-normal text-base leading-8 my-5 text-black"
+          dangerouslySetInnerHTML={{ __html: item }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)